website/static/js/myProjects.js

```diff
@@ -523,7 +549,8 @@ var MyProjects = {
         self.loadCollections = function _loadCollections (url){
             var promise = m.request({method : 'GET', url : url, config : xhrconfig});
             promise.then(function(result){
-                result.data.forEach(function(node){
+                self.sorted_collections = sortCollections(result.data);
```

Comment:
Just a drive-by comment: In general (both in Python and JS), avoid setting attributes outside of a constructor. Also, don't set any unnecessary attributes.